.github/workflows/config/spelling_allowlist.txt

```diff
@@ -263,6 +267,7 @@ lossy
 lvalue
 makefiles
 merchantability
+mgmn
```

Comment:
Code artifacts should be quoted in backquotes (or whatever is appropriate) rather than added as English words here.

Reply:
Can you revisit all of these words being added? Some others look suspiciously like code artifacts to me. Thanks.